warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionsTest.java

```diff
@@ -272,13 +290,14 @@ public void testEvaluationNoContentFields() {
         list1 = asList(Arrays.asList(1, 2, 3), Arrays.asList(0, 0, 0));
         list2 = asList(Arrays.asList(5, 6, 7), Arrays.asList(0, 0, 3)); // match (7-3_ should match (3+1)
         
-        termOffSetMap.put("dog", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", false, eventId), list1)));
-        termOffSetMap.put("cat", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", false, eventId), list2)));
+        termOffSetMap.putTermFrequencyList("dog", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", false, eventId), list1)));
+        termOffSetMap.putTermFrequencyList("cat", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", false, eventId), list2)));
         
         context.set(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME, termOffSetMap);
         Object o = expr.evaluate(context);
         
         Assert.assertTrue(expect(o, false));
+        assertPhraseOffsetsEmpty();
```

Comment:
good

Reply:
the within function is always tricky as it always seems to be 1 off.  3,4 is actually a distance of 2 in this regard.

Reply:
I am guessing that the within distance has to include the entire range of 7-3 to 7+3.